rpc_jobs/standard_job_postmerge.yml

```diff
@@ -167,10 +167,7 @@
             user = 'SYSTEM'
           }}
           if(user == 'SYSTEM' && JIRA_PROJECT_KEY != ''){{
-            common.create_jira_issue(JIRA_PROJECT_KEY,
-                                     env.BUILD_TAG,
-                                     env.BUILD_URL,
-                                     "Task")
+            common.build_failure_issue("RE")
```

Comment:
Shouldn't this be `JIRA_PROJECT_KEY`, instead of `RE`?

Reply:
Yep, done.